### PR TITLE
Add multichain multicall addresses

### PR DESCRIPTION
**Pull request by jdowning100**
For this wallet to work properly, the [Multicall](https://github.com/mds1/multicall/blob/main/src/Multicall3.sol) Solidity smart contract needs to be deployed in every zone and the corresponding address needs to be updated in each respective zone's `multicall` variable in background/constants/networks.ts
This needs to be done every time the chain is wiped, so should be a part of CI/CD.

### CODE REVIEW

File: background/main.ts [hunk 1]
```diff
@@ -527,6 +529,52 @@ export default class Main extends BaseService<never> {
     globalThis.main = this
   }
 
+  async startBalanceChecker(): Promise<void> {
```
Comment (wizeguyy) [inline background/main.ts:532]: This was already added in your other PR, right? Does this PR needs a rebase?

File: background/constants/networks.ts [hunk 2]
```diff
@@ -431,55 +432,64 @@ export const DEFAULT_QUAI_LOCAL = {
       name: "Cyprus One",
       shard: "cyprus-1",
       rpc: "http://localhost:8610",
-      blockExplorerUrl: "http://localhost:4002"
+      blockExplorerUrl: "http://localhost:4002",
```
Comment (gameofpointers) [inline background/constants/networks.ts:435]: why is this localhost? and not using blockscout url?

File: ui/pages/Settings/SettingsAddCustomAsset.tsx [hunk 3]
```diff
@@ -204,6 +204,24 @@ export default function SettingsAddCustomAsset(): ReactElement {
       await dispatch(setSnackbarMessage(t("snackbar.success")))
       setIsImportingToken(false)
       history.push("/")
+      await dispatch( // TODO: Only do this if necessary
```
Comment (gameofpointers) [inline ui/pages/Settings/SettingsAddCustomAsset.tsx:207]: what does this TODO mean?

**Review by wizeguyy — COMMENTED** [review]
I think it needs a rebase, and I also second @gameofpointers's comments, but otherwise looks good

File: background/main.ts (hunk 1 above)
Comment (jdowning100) [reply background/main.ts:532]: My first PR wasn’t merged, I think this includes commits from the other PR

File: background/constants/networks.ts (hunk 2 above)
Comment (jdowning100) [reply background/constants/networks.ts:435]: DEFAULT_QUAI_LOCAL is localhost

File: ui/pages/Settings/SettingsAddCustomAsset.tsx (hunk 3 above)
Comment (jdowning100) [reply ui/pages/Settings/SettingsAddCustomAsset.tsx:207]: importCustomToken() is supposed to update account balances but it doesn’t for some weird reason. So in the future we shouldn’t have to call updateAccountBalance after importCustonToken.

File: background/main.ts (hunk 1 above)
Comment (jdowning100) [reply background/main.ts:532]: To be clear, I rebased this PR on top of the other one.

File: background/constants/networks.ts (hunk 2 above)
Comment (wizeguyy) [reply background/constants/networks.ts:435]: Shouldn't it be `"https://dev.cyprus1.quaiscan.io"`, just like all the others?

File: ui/pages/Settings/SettingsAddCustomAsset.tsx (hunk 3 above)
Comment (gameofpointers) [reply ui/pages/Settings/SettingsAddCustomAsset.tsx:207]: okay as long as we know to change it

**Comment by gameofpointers** [comment]
Closed here https://github.com/PelagusWallet/pelagus/pull/10

**Pull request closed without merging**